nbs/common.base_recurrent.ipynb

```diff
@@ -50,10 +50,12 @@
     "import random\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.            # TODO: clean hack and add correct use of self.scaler
</pre><p>Should we open a  issue? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
This is the issue of the pending inclusion of the `scaler_type` input to RNN-based models
https://github.com/Nixtla/neuralforecast/issues/295